akka-distributed-data/src/main/scala/akka/cluster/ddata/DistributedData.scala

```diff
@@ -45,7 +40,20 @@ class DistributedData(system: ExtendedActorSystem) extends Extension {
       system.log.warning("Replicator points to dead letters: Make sure the cluster node is not terminated and has the proper role!")
       system.deadLetters
     } else {
-      val name = config.getString("name")
-      system.systemActorOf(Replicator.props(settings), name)
+      system.systemActorOf(Replicator.props(settings), ReplicatorSettings.name(system, None))
     }
+
+  /**
+   * Returns true if this member is not tagged with the role configured for the
+   * replicas.
+   */
+  private[akka] def isTerminated: Boolean =
```

Comment:
this was public so it has to remain public

Reply:
You are correct, and I must have been thinking I was in the typed DD when I wrote that, strange and thanks for catching it!